test/unit/BPool.t.sol

```diff
@@ -818,230 +818,6 @@ contract BPool_Unit_GetSpotPriceSansFee is BasePoolTest {
   }
 }
 
-contract BPool_Unit_SwapExactAmountIn is SwapExactAmountInUtils {
```

Comment:
btw I plan on taking care of the `BCoWPool.verify` tests, which rely on `SwapExactAmountInUtils` next, so we can get rid of the abstract contract as well

Reply:
actually i saw today that many of the Utils.sol methods are unused, so we should make a cleaning after getting the final coverage

Reply:
making a list in BAL-155